ckanext/basiccharts/plugin.py

```diff
@@ -122,28 +136,22 @@ class BasicGrid(p.SingletonPlugin):
 
     p.implements(p.IConfigurer, inherit=True)
     p.implements(p.IResourceView, inherit=True)
-    p.implements(p.ITemplateHelpers)
+    p.implements(p.ITemplateHelpers, inherit=True)
 
     def get_helpers(self):
-        return {'view_data': _view_data}
+        return {'view_data': _view_data,
+                'basicgrid_ckan_version': Version,
```

Comment:
Why doesn't this follow the same convention of creating a static method and calling `version_builder`?
Or why doesn't the other definition of something similar do the same as is done here?